<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetThumbnailOptionsDialog</class>
 <widget class="QDialog" name="SetThumbnailOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>564</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="Result_groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>First filename:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="firstFilename">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>filename</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PartOfPath_label">
        <property name="text">
         <string>Part of path:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="PartOfPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>PartOfPath</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Event_label">
        <property name="text">
         <string>Event label:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="EventLabel">
        <property name="text">
         <string>Event label</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="DirectoryStructure_groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Directory structure</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Level 1 (Basis):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Level1_lineEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>static</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>frist position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>last position</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Level 2 (Campaign):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Level2_lineEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="Level2_first_spinBox"/>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="Level2_last_spinBox"/>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Level 3 (Site):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="Level3_lineEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="Level3_first_spinBox"/>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="Level3_last_spinBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Level 4 (Event):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="Level4_lineEdit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="Level4_first_spinBox"/>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="Level4_last_spinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="ThumbnailSize_groupBox">
     <property name="title">
      <string>Size of thumbnail</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="Width_spinBox"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="Height_spinBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Turn images</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Angle:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="TurnAngle_spinBox">
        <property name="maximum">
         <number>359</number>
        </property>
        <property name="singleStep">
         <number>90</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>degree clockwise</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="Options_groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="CreateThumbnails_checkBox">
        <property name="text">
         <string>create thumbnails</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="TurnImages_checkBox">
        <property name="text">
         <string>turn images</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="UploadThumbnails_checkBox">
        <property name="text">
         <string>upload thumbnails</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="UploadImages_checkBox">
        <property name="text">
         <string>upload images</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="RunScript_checkBox">
        <property name="text">
         <string>run script</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Level1_lineEdit</tabstop>
  <tabstop>Level2_lineEdit</tabstop>
  <tabstop>Level2_first_spinBox</tabstop>
  <tabstop>Level2_last_spinBox</tabstop>
  <tabstop>Level3_lineEdit</tabstop>
  <tabstop>Level3_first_spinBox</tabstop>
  <tabstop>Level3_last_spinBox</tabstop>
  <tabstop>Level4_lineEdit</tabstop>
  <tabstop>Level4_first_spinBox</tabstop>
  <tabstop>Level4_last_spinBox</tabstop>
  <tabstop>Width_spinBox</tabstop>
  <tabstop>Height_spinBox</tabstop>
  <tabstop>TurnAngle_spinBox</tabstop>
  <tabstop>CreateThumbnails_checkBox</tabstop>
  <tabstop>UploadThumbnails_checkBox</tabstop>
  <tabstop>TurnImages_checkBox</tabstop>
  <tabstop>UploadImages_checkBox</tabstop>
  <tabstop>RunScript_checkBox</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
